docs/openapi/schemas/Verification.yml

```diff
@@ -4,15 +4,16 @@ properties:
   title:
     type: string
     enum: 
-      - Proof
       - Activation
       - Expired
```

Comment:
`Activation` should be paired with `Expiration` and `Revocation` ... 
or `Expired` should be paired with `Activated` and `Revoked`.

`Proof` and `Signature` might also be better tweaked to 
`Proved` and `Signed`...

Reply:
Thanks for the feedback, @TallTed. I've just created #481 to discuss this sort of thing, would you mind commenting there so we can keep this focused on making the property required?

Reply:
Done